portfolio/src/main/webapp/style.css

```diff
@@ -8,6 +8,14 @@
   margin-top: 20px;
 }
 
-img {
+/* img {
   border-radius: 50%;
+} */
```

Comment:
oops, this shouldn't be here. Will delete because I can just use the bootstrap image.round class